.prettierrc.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "trailingComma": "es5"
```

Comment:
The [new default for this setting]() created a lot of churn [in 17 files, for trailing commas](https://github.com/camunda/camunda-platform-docs/actions/runs/6041539288/job/16394708461), in situations where I don't personally care very much. Files like the sidebars were especially hit hard -- with hundreds of changed lines, to add trailing commas. 

The alternative to setting this configuration here would be to accept all the trailing comma changes. If that's preferred, I can do that; I personally don't care too much about it, and chose to add this config setting because it resulted in fewer changed files.

@akeller let me know if you disagree with adding this config setting!

Reply:
I had to dig a bit to realize what the default was, so linking it here - https://prettier.io/docs/en/options.html#trailing-commas (`all`). I updated my local docs copy, ran `npm run format` and did an `npm run build` and took a little peek while the build successfully ran.

I know there are discrepancies in the sidebars files with trailing commas, but it is the difference between `.js` and `.json` (the versioned sidebars).

`sidebars.js` has commas everywhere. I was shocked any could possibly be missing, but I guess they were!

Given all this, I also don't have any strong feelings either, so I agree with your choice here.